Sources/MongoSwift/MongoCollection+Read.swift

```diff
@@ -79,7 +79,7 @@ extension MongoCollection {
      *
      * - Returns: The count of the documents that matched the filter
      */
-    public func countDocuments(_ filter: Document = [:], options: CountDocumentsOptions? = nil) throws -> Int {
+    private func countDocuments(_ filter: Document = [:], options: CountDocumentsOptions? = nil) throws -> Int {
```

Comment:
why were these made private?

Reply:
they aren't actually implemented, so we figured it was better to hide them rather than have the user find out at runtime via an error.